source/server/http/admin.h

```diff
@@ -118,6 +120,7 @@ class AdminImpl : public Admin,
   Http::Code handlerServerInfo(const std::string& url, Buffer::Instance& response);
   Http::Code handlerStats(const std::string& url, Buffer::Instance& response);
   Http::Code handlerQuitQuitQuit(const std::string& url, Buffer::Instance& response);
+  Http::Code handlerListenerAddresses(const std::string& url, Buffer::Instance& response);
```

Comment:
Might want to make this more generic, e.g. `handlerListenerInfo`, to allow later extension for more than addresses.